### ISSUE TITLE
Seeking Maintainers

### ISSUE THREAD

**Issue by oyejorge** [orchidjs/tom-select#797]
### Bug description

I no longer have time to maintain Tom Select

### Expected behavior

We need one (or more) experienced TS developers with spare time to take over

### Steps to reproduce

Start open source project, invest time, work/life take over, os project suffers...

### Additional context

Who made this issue tracker anyways!

**Comment by nwalters512** [comment]
👋 I use this library and IMO it's the best option for someone not looking to drag in React/etc. While I candidly can't commit huge amounts of time (so I definitely won't be doing wholesale refactors/rewrites), I'd love to help fix up small bugs.

I'm currently facing https://github.com/orchidjs/unicode-variants/issues/2, which is a blocker for me since it's a transitive dependency of `tom-select`. That's a separate project, but still under the same org, so I'm hopeful that I could contribute fixes there as well.

Could you clarify what level of involvement you're looking for in a maintainer? That is, if I can submit PRs, test my own code, prove that it's working as intended, and so on, are you willing to rubber-stamp, merge, and publish to npm? Or are you looking for someone to take full repo/npm permissions? I'd prefer the former, but would be willing to do the latter if required.

**Comment by oyejorge** [comment]
Looking for the latter. There are quite a few existing pull requests and issues that need attention.

**Comment by nwalters512** [comment]
Gotcha. It looks like the existing PRs aren't horribly complex. I'd like to volunteer, though hopefully some others can chip in to share the load as well.

**Comment by nwalters512** [comment]
@oyejorge I got the org invite - thanks! How do you want to handle npm publishes?

**Comment by nwalters512** [comment]
Perfect, thank you. I accepted it. How about for other packages like https://github.com/orchidjs/unicode-variants?

**Comment by nwalters512** [comment]
Many thanks! I should be good now. Assuming there's no one else to review my own changes, I'll go ahead and start doing things, sound good? Hopefully I manage to not break anything 🙂

**Comment by rajraj** [comment]
This is awesome. I use tom-select and its one of the best libraries out there. I am not a js/ts expert but will keep an eye on issues and see if I can contribute. Thanks for taking this on @nwalters512

**Comment by sb-relaxt-at** [comment]
I agree, from my point of view it is one of the best libraries for fancy selects. Thanks @nwalters512 for taking over and thanks @oyejorge for all the effort you put into it.

**Comment by JackAtOmenApps** [comment]
@oyejorge Thank you for creating this library! I've enjoyed using it in my projects for nearly 2 years ❤️

@nwalters512 Thank you for taking this on! I am glad we can continue using Tom Select 😊

**Comment by JSteitz** [comment]
> Gotcha. It looks like the existing PRs aren't horribly complex. I'd like to volunteer, though hopefully some others can chip in to share the load as well.

I can support you with this.
I've been looking for a project that I can contribute to for some time now.

**Comment by KES777** [comment]
I started to use this component recently a lot, but I am not sure I will have enough time to maintain it fully. Probably I could participate at PRs discussion and provide general feedback.
No any promises.
@oyejorge

**Comment by ericscales** [comment]
I'd love to assist.

**Comment by oyejorge** [comment]
Ok, let's get serious about finding a/some new maintainer(s). If you'd like to take over, please show your interest/commitment by sharing your own forked Tom Select repo. Please include a short list of updates you made to the codebase and a link to your published package on  [npmjs](https://www.npmjs.com/).

**Comment by ericscales** [comment]
> Ok, let's get serious about finding a/some new maintainer(s). If you'd like to take over, please show your interest/commitment by sharing your own forked Tom Select repo. Please include a short list of updates you made to the codebase and a link to your published package on [npmjs](https://www.npmjs.com/).

LOL.  I have none of that.  There are plenty of pull requests to go through and I'd just like to get some of those reviewed and merged.

**Comment by nwalters512** [comment]
@ericscales if you'd like to work with contributors to get their PRs into a mergeable state and tested, I can do the final review/merge/release.

**Comment by KES777** [comment]
As far, as I can see here are people who actively using this component and want some issues to be fixed. They created PRs. But because no maintainers are here PRs stale.

The solution to this would be:
1. Implement feature/fix
2. Add some tests
3. Other active members review that PR. 3 from 5 reviews are required.
4. Once condition met -- PR is merged.

**Comment by ericscales** [comment]
@KES777 - yes, I don't understand it.  I understand not having the time to maintain a repo, and have the utmost respect for everyone that contributed to this amazing component.  However, I don't understand the point of this thread if the current maintainers that need help maintaining the repo, are going to seek help and then add a bunch of restrictions and caveats months later.

There are many people here who had offered to help.  Is it all going to be perfect, no; but, let us try and help in our free time.

**Comment by kevinpapst** [comment]
I am glad that it is handled this way. Tom-Select is downloaded tenth of millions times every month and you cannot simply invite everyone who leaves a comment to become a contributor with merge permissions in such a widely used project. Thank you @oyejorge for handling it responsible.

**Comment by KES777** [comment]
@kevinpapst People did theirs PRs here and want to continue contribute/investigate their time. So @oyejorge can review them and decide is it was done good then he can grant merge permission.

Otherwise the steam will go out and `such a widely used project with tenth of millions times downloads every month` just die. IMHO. This is very irresponsible.

Example: https://github.com/orchidjs/tom-select/pull/905
Why this should not be merged? May you grant that person review permission? You still can restrict merges to `code owners` and go through PRs with 3-5 reviews bi-weekly for the first time. After 6mon you can grant merge permission to some people with many reviews. Let's get it forward! Do not ask people to do something abstract.

**Comment by ericscales** [comment]
> [@ericscales](https://github.com/ericscales) if you'd like to work with contributors to get their PRs into a mergeable state and tested, I can do the final review/merge/release.

Fair enough.  I commented on a number of PRs to ask what was needed to get them merged.

**Comment by robinsingh1** [comment]
Hi @oyejorge , would be interested in helping maintain the project - what could potential next steps be . Would love to help out

**Comment by xJuvi** [comment]
Hey @oyejorge 
I'm also able to Support and maintain this project. I use tom-select in a private project and found some bugs in the past. But i'm unhappy that Open PRs are'nt checked and merged. 

Should i help to review them?

Kind regards
Hannes
